extensions/ql-vscode/package.json

```diff
@@ -364,6 +366,11 @@
           "dark": "media/dark/edit.svg"
         }
       },
+      {
+        "command": "codeQLDatabasesExperimental.setSelectedItem",
+        "title": "Select Item",
+        "icon": "$(circle-small-filled)"
```

Comment:
This icon isn't ideal, but it's small and close enough to the "[file decoration](https://github.com/github/vscode-codeql/pull/1840/files#diff-61f334e838ad96942f9e0a876a8e4523b152a96947637e4e6a7489297c51ed64R16)" that we display when an item is selected.